### DISCUSSION TITLE
[Stundenplan] Als Schüler will ich eine E-Mail bekommen falls sich etwas am Stundenplan ändert, damit ich auf dem neusten Stand bin und informiert bin falls eine Stunde ausfällt.

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#16] [User Stories]
### Beschreibung:
Wenn der etwas am Stundenplan geändert wird soll automatisch eine Benachrichtigung per E-Mail und oder der Website an alle Teilnehmer gesendet werden, welche die Änderung enthält.

### Akzeptanz-Kriterien:

- [ ] Bei einer Änderung bekommt der Schüler eine Nachricht
- [ ] Die Nachricht enthält was sich geändert hat und wann 

### Issue
- #97 
- #239

**Comment by Schmarvinius** [comment]
Finde per Notification ist conditional und per E-Mail sollte optional sein.
@Kronprinz03

**Reply by Kronprinz03** [reply]
Hm finde es halt ziemlich wichtig das man benachrichtet werden

**Comment by two-Gee** [comment]
nicht mehr in requirements
